src/snowflake/snowpark/_internal/proto/update-from-local.sh

```diff
@@ -15,7 +15,7 @@ elif [ -n "$1" ]
 fi
 set -euxo pipefail
 
-SCRIPT_DIR=$(dirname "$0")
+SCRIPT_DIR=$(dirname "$0")/generated
```

Comment:
This script is now invalid. There's no more make-ast. We need to run `bazel build //Snowpark/ast:py_ast`.